packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -178,7 +178,13 @@ module.exports = {
       //   include: paths.appSrc,
       // },
       {
-        test: /(^|\/)lodash(\/|\.|$)/,
+        test: bpkReactScriptsConfig.amdExcludes
+          ? new RegExp(
+              `(^|/)(${bpkReactScriptsConfig.amdExcludes
+                .concat('lodash')
+                .join('|')})(/|.|$)`
+            )
+          : () => false,
```

Comment:
I could be wrong, but wouldn't this indicate that lodash is only excluded if `amdExcludes` is defined, and therefore it would be a breaking change for any consumers relying on lodash being excluded by default?

Reply:
absolutely! good catch

Reply:
@shaundon i've fixed, can you have another look please? 🙏